markdown.dtx

```diff
@@ -9225,6 +9225,406 @@ defaultOptions.texMathDollars = false
 %</lua,lua-cli>
 %<*manual-options>
 
+#### Option `texMathDoubleBackslash`
+
+`texMathDoubleBackslash` (default value: `false`)
+
+% \fi
+% \begin{markdown}
+%
+% \Optitem[false]{texMathDoubleBackslash}{\opt{true}, \opt{false}}
+%
+:    true
+
+     :  Enable the Pandoc `tex_math_double_backslash` syntax extension.
+
+        ```
+        inline math: \\(E=mc^2\\)
+
+        display math: \\[E=mc^2\\]
+        ```
+
+:    false
+
+     :  Disable the Pandoc `tex_math_double_backslash` syntax extension.
+
+% \end{markdown}
+% \iffalse
+
+##### Lua Module Example {.unnumbered}
+
+Using a text editor, create a text document named `document.tex` with the
+following content:
+``` tex
+\input markdown
+\input lmfonts
+\directlua{
+  local markdown = require("markdown")
+  local newline = [[^^J^^J]]
+  local convert = markdown.new({texMathDoubleBackslash = true})
+  local input =
+    [[\\(E=mc^2\\)]] .. newline .. newline ..
+    [[\\[\hat{f} \left ( \xi  \right )= \int_{-\infty}^{\infty} f\left ( x  \right ) e^{-i2\pi \xi x} dx\\]]]
+  tex.sprint(convert(input)) }
+\bye
+```````
+Then, invoke LuaTeX from the terminal:
+``` sh
+luatex document.tex
+```````
+A PDF document named `document.pdf` should be produced and contain the
+following text:
+
+> \\(E=mc^2\\)
+>
+> \\[\hat{f} \left ( \xi  \right )= \int_{-\infty}^{\infty} f\left ( x  \right ) e^{-i2\pi \xi x} dx\\]
```

Comment:
Either rewrite to a form that will be recognized by the default extensions of Pandoc (here and hereafter), or add the `tex_math_double_backslash` extension to [`Makefile`][1].

 [1]: https://github.com/Witiko/markdown/blob/36290d5c75687462734a3d7fac29b4bdd90659a5/Makefile#L130